media-sound/ladish/ladish-1.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit python-single-r1 waf-utils
+
+DESCRIPTION="LADI Session Handler - a session management system for JACK applications"
+HOMEPAGE="https://ladish.org"
+
+SRC_URI="https://dl.ladish.org/ladish/${P}-g36c489e4.tar.bz2"
+S="${WORKDIR}/${P}-g36c489e4"
+
+LICENSE="GPL-2"
```

Comment:
I think it should be `GPL-2+`, i've checked some source files and they all say:
```
 * LADI Session Handler is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
```

Reply:
common/klist.h is linked list code borrowed from linux kernel, and thus is GPL2 only

Reply:
I see, well in that case we should include both, like: `LICENSE="GPL-2 GPL-2+"`

Reply:
Fixed in the new version